schema/proto3/neighbor.proto

```diff
@@ -43,6 +43,9 @@ message NeighborConfiguration {
         NeighborType neighbor_type = 1;
         string subnet_id = 2;
         string ip_address = 3;
+        string arion_group = 4;
```

Comment:
@DavidLiu506 Let us discuss this. Hopefully we could reuse Alcor basic data structure to support Arion.